src/EPR.Calculator.API/Validators/CreateLapcapDataValidator.cs

```diff
@@ -11,6 +11,8 @@ public CreateLapcapDataValidator()
             RuleFor(x => x.ParameterYear).NotEmpty().WithMessage((ErrorMessages.YearRequired));
             RuleFor(x => x.LapcapDataTemplateValues).NotNull().Must(x => x.Count() == LapcapDataUniqueReferences.UniqueReferences.Length)
                 .WithMessage((ErrorMessages.LapcapDataTemplateValuesMissing));
+            RuleFor(x => x.LapcapFileName).NotEmpty().WithMessage(ErrorMessages.FileNameRequired);
+            RuleFor(x => x.LapcapFileName).MaximumLength(256).WithMessage(ErrorMessages.MaxFileNameLength);
```

Comment:
Can we create a readonly variable and name it MaxFileNameSupported?

Reply:
hi Uday thanks where this variable needs to be created and what value it can have ?

Reply:
done that